org.eclipse.buildship.ui.test/src/main/groovy/org/eclipse/buildship/ui/SWTBotTestHelper.groovy

```diff
@@ -0,0 +1,94 @@
+package org.eclipse.buildship.ui
```

Comment:
Donat and Etienne technically wrote most of the code. Should I add them to the copyright header as well?